plans/install/docs.fmf

```diff
@@ -6,6 +6,7 @@ prepare:
     how: install
     package: python3-pip
 execute:
+    how: tmt
```

Comment:
@happz no more default `how` ?

Reply:
That's correct, Added on today's hacking meeting notes.